articles/openshift/howto-using-aro.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Using `az aro` | Microsoft Docs
+description: Create, access, and delete clusters with `az aro` extension
+author: klamenzo 
+ms.author: b-lejaku
```

Comment:
check this?

Reply:
I think we'll put @sakthi-vetrivel or another MS author's id there.